### PR TITLE
Can use requesting message to send ether if recipient name contains double quotes (#1411)

### LINKED ISSUE #1411: Can't use requesting message to send ether if recipient name contains double quotes [send]
### Description

*Type*: Bug

*Summary*: 
1. User with name ```Name with "double quotes"``` sends request to user A. User A can't send ether by tapping on this request because there is no send button shown:
screenshot_20170704-173026

2. User with name ``` Name with "double quote```  sends request to user A. User A can't send ether because on sending it there is error shown "Amount is not a valid number"
screenshot_20170704-173542

#### Expected behavior
User A can send ether to user B who sent a request if username of B contains double quotes
#### Actual behavior
User A can't send to user B ether because in one case there is no send button and in another error message is shown

### Reproduction
- As user with name ```Name with "double quotes"``` request from user A 0.1 ether
- As user A: open chat with user ```Name with "double quotes"```, tap on received request -> try to tap on send button if it's shown (in my case it's not)
- As user ``` Name with "double quote``` request from user A 0.2 ether
-  As user A: open chat with ``` Name with "double quote``` , tap on received request and tap on send button. 

### Additional Information
* Status version: 0.9.8-59-gda60a65b+
* Operating System: 
user A: Real device Samsung Galaxy S7, Android 7.0
user with double quotes in name: Real device iPhone 6, iOS 10.2.1

### PR DESCRIPTION
**Pull request by alwx**
Fixes #1411

### CODE REVIEW

File: src/status_im/chat/models/input.cljs [hunk 1]
```diff
@@ -103,13 +103,17 @@
   Input:  ['/send' 'Jarrad' '1.0']
   Output: '/send Jarrad 1.0'
 
+  Input:  ['/send' '\"Jarrad\"' '1.0']
+  Output: '/send Jarrad 1.0'
```
Comment (jeluard) [inline src/status_im/chat/models/input.cljs]: Now that we have tests running, this function looks like a good candidate for unit testing. Actually we already have some tests for the associated `split-command-args` function.

Comment (alwx) [inline src/status_im/chat/models/input.cljs]: Yes, I actually wanted to create tests for every function in `chat/models/input.cljs`. Will probably do it during the weekend or even earlier.

**Comment by rasom** [comment]
@alwx are you going to add tests in this PR?

**Comment by alwx** [comment]
@rasom no. In this case it's a good idea to add tests for all input methods, but it should be done separately

**Pull request merged**